.github/workflows/lint.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           key: venv-${{ env.PYTHON_VERSION }}-${{ runner.os }}-${{ hashFiles('**/pyproject.toml', '**/poetry.lock') }}
 
       - name: Install dependencies and project in dev mode
-        if: steps.cache.outputs.cache-hit != true
+        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
Why did you have to do this? It looks silly

Reply:
@igor-sb I am trolling my sibling

Reply:
See PR #8 , basically if quotes aren't added,  'true' won't register as a boolean, so this would always evaluate to false, and the cached virtual environment would never be detected, so dependencies would always have to be reinstalled from scratch

Reply:
Okay, so true is a non-defined variable, whereas 'true' make the logic check a boolean check?

Reply:
yes, that is my understanding.

Reply:
Yeah 'true' would be a string. It seems that when you pass an invalid variable it makes the whole thing false. (I guess it slips through the error checks.)